src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+const Married = ({ person }) => {
+  if (person.isMarried === false) {
+    return <p>I am not married</p>;
+  }
+
+  if (person.sex === 'f') {
+    return <p>{person.partnerName} is my husband</p>;
+  }
+
+  return <p>{person.partnerName} is my wife</p>;
+};
```

Comment:
better to write it directly in Person, using ternary operators

Reply:
I agree, I already tried to do this but there was one error in the linter when I used two ternary operators (? : ? : )there was something like a wrong rule, although everything worked

Reply:
That's why I put it in a function so that there is no error in the linter

Reply:
> better to write it directly in Person, using ternary operators

is this error - 12:8  error  Do not nest ternary expressions  no-nested-ternary

Reply:
you can just move some condition in variable and reuse in ternary operator